### PR TITLE
Error handling added and refactored code

### PR DESCRIPTION
**Pull request by sidsbrmnn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,12 +21,12 @@ BASE_URL = http://localhost:3000
...
-cloud_name = <cloud_name_here>
-api_key = <api_key_here>
-api_secret = <api_secret>
+```bash
+DB_URL='mongodb://<dbuser>:<dbpassword>@ds111476.mlab.com:11476/<db_name>'
+BASE_URL='http://localhost:3000'
+cloud_name='<cloud_name_here>'
+api_key='<api_key_here>'
+api_secret='<api_secret>'
 ```
```
Comment (AdithyaBhat17) [inline README.md:30]: I faced issues when i put quotes for string literals in the env. Are you sure this works?

File: README.md [hunk 2]
```diff
@@ -37,8 +37,8 @@ api_secret = <api_secret>
 * Open a new terminal window and move to the *ui* folder and install dependencies - `cd ui && npm install`
 * create a `.env` file in the root of `ui/` and add the following inside it
 
-```.env
-REACT_APP_BASE_URL = http://localhost:8080
+```bash
+REACT_APP_BASE_URL='http://localhost:8080'
```
Comment (AdithyaBhat17) [inline README.md:41]: "I faced issues when i put quotes for string literals in the env. Are you sure this works?"

Ignore this if its working fine.

File: services/logging.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+require('express-async-errors');
+
+module.exports = function() {
+  process.on('uncaughtException', error => {
+    console.log(error);
+    process.exit(1);
+  });
```
Comment (AdithyaBhat17) [inline services/logging.js:7]: uncaughtException is going to be deprecated soon. (also it doesn't give complete log info)

**Review by AdithyaBhat17 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sidsbrmnn) [reply README.md:30]: Yep it does. I use it all the time. String either the way

File: services/logging.js (hunk 3 above)
Comment (sidsbrmnn) [reply services/logging.js:7]: It isn't recommended to resume the app on uncaughtException. Check [here](https://nodejs.org/api/process.html#process_warning_using_uncaughtexception_correctlyl)

Comment (AdithyaBhat17) [reply services/logging.js:7]: okay.

**Review by AdithyaBhat17 — APPROVED** [review]

**Pull request merged**